### PR TITLE
Panel header

### PR DESCRIPTION
**Pull request by FilipLeitner**
## Description

New panel header with better and consistent extra-buttons UI
image
image
image
image
image

- translateHs pipe as a standalone

## Related issues or pull requests

https://redmine.ccss.cz/issues/9763

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [x] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [ ] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [ ] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by FilipLeitner** [comment]
Styler had a specific - panel name + layer name which is not directly possible. Its possible to get a layer name into header vi some absolutely positioned element but I didnt like that much. So thats something taht needs to be resolved yet

+ mobile UI. Havent tested taht either

**Comment by jmacura** [comment]
I really like this design change 👍

**Comment by FilipLeitner** [comment]
image

**Comment by FilipLeitner** [comment]
LAyer title in styler like this:
image

Thinking about the X button on the right, wether it should be there or not. Other than that I think its ready

**Comment by FilipLeitner** [comment]
> Probably also btn-outline-primary here, for consistency... image

o missed that one thanks it comming

**Comment by jmacura** [comment]
And what about this jumping panel header?
image

**Comment by FilipLeitner** [comment]
One more thing is to unite adddata and composition scrolling behavior (to have for both asi it is with compositions)  it just doesnt want to cooperate

**Comment by jmacura** [comment]
> LAyer title in styler like this: image
> 
> Thinking about the X button on the right, whether it should be there or not. Other than that I think its ready

Regarding this, I propose to use the new "bg-primary-subtle" as it is less visually intrusive.
image

Not sure about the "x" button either... All the "hidden" nature of Styler panel is kind of non-systematic. Let's keep the close button for now.

**Comment by FilipLeitner** [comment]
> > LAyer title in styler like this: image
> > Thinking about the X button on the right, whether it should be there or not. Other than that I think its ready
> 
> Regarding this, I propose to use the new "bg-primary-subtle" as it is less visually intrusive. image
> 
> Not sure about the "x" button either... All the "hidden" nature of Styler panel is kind of non-systematic. Let's keep the close button for now.

Had this as a second option but opted for just a primary but the title itself is not that importat so I guess this is better indeed. Thanks for suggestions

**Review by jmacura — APPROVED** [review]
In general, I approve. As I said already, I really like the new design.
Pls see the comment regarding in-component styles and my commits.

**Comment by jmacura** [comment]
The information about multi-measurement has been gone from the panel header, so I have exposed the option in the panel UI. Until now, it was only possible to switch modes with the CTRL key 🤪 
image

File: projects/hslayers/src/components/layout/panels/panel-header/panel-header.component.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      }
+
+      @media screen and (max-width: 767px) {
+        .extra-content {
+          padding: 0 1rem 0 1rem;
+          justify-content: space-between;
+          gap: 1rem;
+        }
+      }
+    `,
```
[the remark this answers is not in the document]
Comment (FilipLeitner) [reply projects/hslayers/src/components/layout/panels/panel-header/panel-header.component.ts:68]: Yup this got a bit larger than expected

[the remark this answers is not in the document]
Comment (jmacura) [reply projects/hslayers/src/components/layout/panels/panel-header/panel-header.component.ts]: Great, thanks!

**Pull request merged**